src/WalterModem.h

```diff
@@ -2875,6 +2894,13 @@ class WalterModem
          */
         static char _getLuhnChecksum(const char *imei);
 
+        /**
+         * @brief this function calls all the registerd eventHandler functions.
```

Comment:
Please make the `@brief` description a sentence (with a capital letter at the beginning)